.github/workflows/rebuild-docker-image.yml

```diff
@@ -26,4 +26,4 @@ jobs:
           username: '${{ secrets.IOTALEDGER_HUB_DOCKER_LOGIN }}'
           password: '${{ secrets.IOTALEDGER_HUB_DOCKER_PASSWORD }}'
           tags: latest,${{github.event.inputs.tagName}}
```

Comment:
It shouldn't be tagged as latest if we are rebuilding an older image.

Reply:
```suggestion
          tags: ${{github.event.inputs.tagName}}
```